annotation-file-utilities/src/main/java/org/checkerframework/afu/scenelib/io/IndexFileParser.java

```diff
@@ -486,6 +486,16 @@ private Object parseScalarAFV(ScalarAFT aft) throws IOException, ParseException
           if (type == long.class) {
```

Comment:
Please remove or adjust the comment above "in particular, scientific notation is a problem".